packages/cosmos_ui_components/lib/components/cosmos_divider.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:flutter/material.dart';
+
+class CosmosDivider extends StatelessWidget {
+  const CosmosDivider({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = CosmosTheme.of(context);
+    return Padding(
```

Comment:
I'm pretty sure this padding will differ on different screen, so what I'd do is to pass in the `EdgeInsets` as param with default value of `null`:

```dart
...
final EdgeInsets? edgeInsets;

const CosmosDivider({
    Key? key, 
   this.edgeInsets,
}) : super(key: key);

... 
Widget build(BuildContext context) {
    ...
    padding: edgeInsets ?? EdgeInsets.symmetric(horizontal: theme.spacingL)
    ...
}

```

Reply:
Updated